chat.delta.desktop.tauri.yml

```diff
@@ -0,0 +1,136 @@
+
+      # install the core we build locally instead of the prebuild
+      # patch package.json to exclude the extra architectures that we excluded when building the pnpm store
+      # also exclude electron from allowed install script
+      - |
+        jq ".pnpm.supportedArchitectures.os = [\"linux\"] | .pnpm.supportedArchitectures.cpu = [\"x64\",\"arm64\"] | del(.pnpm.onlyBuiltDependencies[] | select(. == \"electron\"))" package.json > package.new.json
+        mv package.new.json package.json
+
+      # install deps from store
+      - |
+        node tool_replay.mjs &
+        REPLAY_PID=$!
+        env DEBUG="*" pnpm install --prefer-offline --frozen-lockfile
+        kill $REPLAY_PID
+
+      # patching some cargo files
+      # for some reason this is needed as soon as we enable the flatpak feature
+      - mv cargo/config cargo/config.toml
+      - sed -i 's/\\\\"/\"/g' cargo/vendor/tauri-plugin-single-instance/Cargo.toml
+      - sed -i 's/\\\\"/\"/g' cargo/vendor/tao/Cargo.toml
+
+      # build & package
+      - |
+        export CARGO_HOME=/run/build/deltachat-tauri/cargo
+        pnpm --filter @deltachat-desktop/target-tauri tauri build -b deb -- --offline --features=flatpak
+      - ls target/release/bundle
+
+      # extract deb and install it
+
+      - ar x target/release/bundle/deb/*.deb
+      - mkdir deb_data
+      - tar -xvzf data.tar.gz -C deb_data
+      - cp -r deb_data/usr/* /app
+
+      # icons
+      # - install -Dm 0644 deltachat-tauri.svg /app/share/icons/hicolor/scalable/apps/deltachat-tauri.svg
+      - install -Dm 0644 chat.delta.desktop.tauri.appdata.xml /app/share/metainfo/chat.delta.desktop.tauri.metainfo.xml
+      - install -Dm 0644 deltachat-tauri.svg /app/share/icons/hicolor/scalable/apps/chat.delta.desktop.tauri.svg
+
+      # rename files
+      - mv /app/share/icons/hicolor/256x256@2/apps/deltachat-tauri.png /app/share/icons/hicolor/256x256@2/apps/chat.delta.desktop.tauri.png
+      - mv /app/share/icons/hicolor/128x128/apps/deltachat-tauri.png /app/share/icons/hicolor/128x128/apps/chat.delta.desktop.tauri.png
+      - mv /app/share/icons/hicolor/32x32/apps/deltachat-tauri.png /app/share/icons/hicolor/32x32/apps/chat.delta.desktop.tauri.png
+      - mv /app/share/applications/deltachat-tauri.desktop /app/share/applications/chat.delta.desktop.tauri.desktop
+
+      # modify icon path in desktop file
+      - sed -i 's/Icon=deltachat-tauri/Icon=chat.delta.desktop.tauri/; s/Name=deltachat-tauri/Name=Delta Chat Tauri/' /app/share/applications/chat.delta.desktop.tauri.desktop
+      - echo "StartupWMClass=deltachat-tauri" >> /app/share/applications/chat.delta.desktop.tauri.desktop
+
+    sources:
+      # Delta Chat
+      - generated/desktop-git.json
+
+      # dependencies
+      - generated/proxy-registry-cache-manifest.json
+      - generated/sources-rust.json
+      - generated/pnpm.json
+
+      # assets
+      - type: file
+        path: chat.delta.desktop.tauri.256.png
```

Comment:
why is this needed?

Reply:
good catch, it is not needed, I'll remove it